modules/social_features/social_embed/src/SocialEmbedConfigOverride.php

```diff
@@ -9,7 +9,7 @@
 use Drupal\Core\Extension\ModuleHandlerInterface;
 
 /**
- * Class SocialEmbedConfigOverride.
+ * Configuration overrides for Social Embed module.
```

Comment:
Given that we no longer use features, this PR is probably a great moment to replace our config override with a configuration update (and corresponding install hook).

Reply:
That's unfortunately not part of our scope (this needs to be part of 10.3.0) and extending the scope will result in a release blocker.